### PR TITLE
#2171 Revise Content of Partner Self-Report Feature

### PR DESCRIPTION
**Pull request by lewischen-aot**
- Modified the question "Does your partner have a valid Social Insurance Number?" to "Is your partner eligible for a B.C. Services Card?" (Applies to PT and FT applications).
- Modified the help text to "Partners must provide their information to StudentAid BC by logging in with a B.C. Services Card. If they are ineligible for a B.C. Services Card (for e.g. if they do not have a valid SIN), select "No" to this question and provide information on their behalf." (Applies to PT and FT applications).
- Modified the warning text to "If partners cannot access a B.C. Services Card, the student will need to report partner information on their behalf. Please be aware this information will be subject to audit and review by StudentAid BC and may delay a student's application" (Applies to PT and FT applications).
- Renamed the form IO API key name "isYourSpouseACanadianCitizen" to "isYourPartnerAbleToReport" (Applies to PT and FT applications).
 - Corrected content typos "the your".

Screenshot of the help text and the question "Is your partner eligible for a B.C. Services Card?" 
image

Screenshot of the warning text and the question "Is your partner eligible for a B.C. Services Card?" 
image

Screenshot of the partner income page for full-time application highlighting "your" as corrections to previous typos "the your"
image

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/sfaa2021-22.json [hunk 1]
```diff
@@ -16137,13 +16137,13 @@
                   "value": ""
                 }
               ],
-              "content": "<Strong> Please be advised your partner will need to sign in to StudentAid BC and provide additional information.</Strong><br/> \r\nPlease check your email after submitting your application. An email will be sent to you with instructions for your partner to sign in. Your application cannot proceed until this information is received.\r\n<br />",
+              "content": "<Strong> Please be advised your partner will need to sign in to StudentAid BC and provide additional information.</Strong><br/> \nPartners must provide their information to StudentAid BC by logging in with a B.C. Services Card. If they are ineligible for a B.C. Services Card (e.g., if they do not have a valid SIN), select \"No\" to this question and provide information on their behalf.",
```
Comment (andrepestana-aot) [inline sources/packages/forms/src/form-definitions/sfaa2021-22.json:16140]: very minor: I don't think we have a comma after "e.g."

**Review by andrepestana-aot — APPROVED** [review]
LGTM!

File: sources/packages/forms/src/form-definitions/sfaa2021-22.json (hunk 1 above)
Comment (andrewsignori-aot) [reply sources/packages/forms/src/form-definitions/sfaa2021-22.json:16140]: Minor also, based on the AC there is a "for" missing also.
Current.
```
(e.g., if they do not have a valid SIN)
```
Expected
```
(for e.g. if they do not have a valid SIN)
```

File: sources/packages/forms/src/form-definitions/sfaa2021-22.json [hunk 2]
```diff
@@ -16760,7 +16760,7 @@
               "defaultValue": ""
             },
             {
-              "label": "<strong>Will  your partner be at home caring for eligible dependent children on a full-time basis during the your entire study period? </strong>",
+              "label": "<strong>Will your partner be at home caring for eligible dependent children on a full-time basis during your entire study period? </strong>",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/sfaa2021-22.json:16763]: Not part of the PR, just sharing for the benefit of DEVs, the `<strong>` is no longer needed since a global style was introduced that takes care of the same. Adding the `<strong>` or not should not cause any visual difference.

**Review by andrewsignori-aot — APPROVED** [review]
Nove work, looks good 👍

File: sources/packages/forms/src/form-definitions/sfaa2021-22.json (hunk 1 above)
Comment (lewischen-aot) [reply sources/packages/forms/src/form-definitions/sfaa2021-22.json:16140]: I checked multiple places whether there is a comma after "e.g.". Most of them suggest that there should a comma, until this reference shows that "e.g." is not followed by a comma in British English: https://jakubmarian.com/comma-after-i-e-and-e-g/. I think we can agree on "e.g." without the comma and "for" 😎

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @lewischen-aot
